.github/workflows/rust.yml

```
@@ -228,10 +228,16 @@ jobs:
         run: cargo +${{ matrix.toolchain }} publish --all-features --dry-run
 
       - name: Publish
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/head/main' }}
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
```

```suggestion
        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
```

Some other `s` I need to add here? 🙃